Sources/Common/DataModel/Quad/index.js

```diff
@@ -0,0 +1,265 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkCell from 'vtk.js/Sources/Common/DataModel/Cell';
+import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
+import { CellType } from 'vtk.js/Sources/Common/DataModel/CellTypes/Constants';
+import vtkTriangle from 'vtk.js/Sources/Common/DataModel/Triangle';
+import vtkPoints from 'vtk.js/Sources/Common/Core/Points';
+
+// ----------------------------------------------------------------------------
+// Static API
+// ----------------------------------------------------------------------------
+
+export const STATIC = {};
```

Comment:
If there is no exportable static API, then you can leave this out. Were you planning on adding some static methods?